ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -198,7 +198,13 @@ fi
 # fi
 log "Choosing openshift version ${OPENSHIFT_VERSION}"
 
-TAGS="prowci:${CLUSTER_NAME}"
+# add USER_TAGS to help with cloud cos
+TAG_Author=$(echo "${JOB_SPEC}" | jq -r '.refs.pulls[].author // empty' || true)
```

Comment:
Here the default value - `empty` is a string or an empty string ?

Reply:
If there is no value of author, it will return the empty string instead of `null`.

Reply:
Classic OCP cluster is using `""` string rather than `empty` string, I think we can keep consistent.

Reply:
The `empty` string I said is the `""` string

Reply:
```
$ echo "${JOB_SPEC}" | jq -r '.refs.pulls[].authors'                                                        
null
$ TAG_Author=$(echo "${JOB_SPEC}" | jq -r '.refs.pulls[].authors // empty'
$ echo "a->$TAG_Author->a"
a->->a